db/seeds.rb

```diff
@@ -60,16 +60,50 @@ def load_basic_account_types
 
 def load_test_users
   puts "Loading test users..."
-  (1..3).each do |i|
-    @user = Member.create(
+
+  # Open suburb csv
+  source_path = Rails.root.join('db', 'seeds')
+  begin
+    suburb_file = File.open("#{source_path}/suburbs.csv")
+  rescue
+    puts "Warning: unable to open suburbs.csv"
+  end
+
+
+  # rake parameter (eg. 'rake db:seed member_size=10')
+  member_size = ENV['member_size'] ? ENV['member_size'].to_i : 3
+
+  (1..member_size).each do |i|
+    @user = Member.new(
         :login_name => "test#{i}",
         :email => "test#{i}@example.com",
         :password => "password#{i}",
         :tos_agreement => true
     )
-    @user.confirm!
+    @user.skip_confirmation!
```

Comment:
Thanks for looking at my code. Currently when a test user is loaded, content of the confirmation email for each user opens up in a browser. However, <code>skip_confirmation</code> automatically confirm the user and avoid generating thousands of confirmation emails/pages. My rationale is most of the time we ignore the confirmation pages, and if someone wants to test the confirmation email content specifically, it can be tested manually.